pages/linux/reflector.md

```diff
@@ -0,0 +1,11 @@
+# reflector
+
+> Arch script to fetch and sort mirrorlists.
+
+- Get all mirrors, sort for speed and save them:
+
+`sudo reflector --sort rate --save /etc/pacman.d/mirrorlist`
```

Comment:
What precisely is meant by speed? Is it _download speed_, _lowest latency_, or something else?

Also, could you wrap `/etc/pacman.d/mirrorlist` in `{{` `}}` please? This lets the user know that they can edit it (it's special syntax supported by tldr clients).

Reply:
Thanks for the friendly tips, I fixed it. 
I'm new to all of this, so thanks for your patience